### PR TITLE
Update contact email content

### PR DESCRIPTION
**Pull request by fsoffel**
# Pull Request \<Title>
`<Description here>`
## Things to do during Peer Review
Please check all boxes before the Pull Request is merged. In case you skip a box, describe in the PRs description (that means: here) why the check is skipped.
- [x] The Github Actions (including Sonarqube Gateway and Lint Checks) are green. This is enforced by Github. 
- [ ] A peer-review has been executed
  - [ ] The code has been manually inspected by someone who did not implement the feature
- [ ] The PR actually implements what is described in the JIRA-Issue
- [ ] At least one test exists testing the new feature
  - [ ] If you have created new test files, make sure that they are included in a test container and actually run in the CI
- [ ] Documentation is updated as required
- [ ] The automated deployment is updated if required
- [ ] If there was a database entity class added, there must also be a migration script for creating the corresponding database if flyway is already used by the service
- [ ] IF there are changes done to the framework data models or to a database entity class, the following steps were completed in order
  - [ ] The version of main currently active on prod is deployed to a dev server with `Reset non-user related Docker Volumes & Re-populate` turned on
  - [ ] It's verified that the CD run is green
  - [ ] The data from prod is migrated via DB dumps (see Wiki page on "Server Migration" for details) 
  - [ ] The feature branch is deployed to the same server with `Reset non-user related Docker Volumes & Re-populate` turned off
  - [ ] It's verified that the CD run is green
  - [ ] The new feature is manually used/tested/observed on the dev server
- [ ] ELSE, the new version is deployed to the dev server "dev1" using this branch
  - [ ] Run with setting `Reset non-user related Docker Volumes & Re-populate` turned on 
  - [ ] It's verified that this version actually is the one deployed (check gitinfo for branch name and commit id!)
  - [ ] It's verified that the CD run is green
  - [ ] The new feature is manually used/tested/observed on dev server
- [ ] Make sure that pull requests in the development tools repository, which are connected to this pull requests, 
are merged

### CODE REVIEW

File: dataland-community-manager/src/main/resources/email-templates/claim_ownership.html.ftl [hunk 1]
```diff
@@ -4,6 +4,8 @@
 <html>
```
Comment (emanueldfine) [inline dataland-community-manager/src/main/resources/email-templates/claim_ownership.html.ftl:4]: Reintegrate the senders and receivers evaluation process with "exit" statement if an EmailBuilder has invalid receivers or cc-receivers.

File: dataland-community-manager/src/main/resources/email-templates/claim_ownership.html.ftl [hunk 2]
```diff
@@ -135,19 +130,13 @@
...
     <tr>
-        <td style="text-align: left; padding:0; margin:0; border: 0; height: 54px; width: 26px">
-            <img src="${baseUrl}/images/email/action_button_left_img.png" width="26" height="54" alt=""
-                 style="border: 0 none; height: auto; line-height: 100%; display:block; outline: none; text-decoration: none;">
-        </td>
+        <td style="text-align: left; padding:0; margin:0; border: 0; height: 54px; width: 26px"></td>
         <td style="background-color: #ff5c00; text-align: center; padding:0; margin:0; border: 0; height: 54px; width: 468px;">
             <a href="${baseUrl}/companies/${companyId}" target="_blank" style="border: 0 none; line-height: 30px; color: #ffffff; font-size: 18px; width: 100%; display: block; text-decoration: none;">
-                CALL TO ACTION
+                REGISTER AND CLAIM OWNERSHIP
```
Comment (emanueldfine) [inline dataland-community-manager/src/main/resources/email-templates/claim_ownership.html.ftl:136]: Wenn man das anklickt und im Cockpit landet, sollte dann nicht ein prominentes Banner (z.B. das was wir schon haben) erscheinen, um direkt zu zeigen wo man ownership claimt, und dann auf die register-Seite redirecten?

Nur ein Verbesserungsvorschlag.

Comment (emanueldfine) [reply dataland-community-manager/src/main/resources/email-templates/claim_ownership.html.ftl:136]: Mit Siegfried besprochen:   Ticket sollte angelegt werden, um die User-Journey - wenn man auf diesen Button hierin der Mail klickt - intuitiver zu gestalten.

Sobald diese Story angelegt wurde, schließe ich diese Konversation.

File: dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/utils/DataRequestManagerUtils.kt [hunk 3]
```diff
@@ -28,6 +28,9 @@ class DataRequestManagerUtils(
     private val leiRegex = Regex("^[0-9A-Z]{18}[0-9]{2}$")
     private val permIdRegex = Regex("^\\d+$")
 
+    private val emptyMutableListOfStoredDataRequestMessageObjects
```
Comment (emanueldfine) [inline dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/utils/DataRequestManagerUtils.kt:31]: misleading Variablenname.
Es handelt sich um eine String-Repräsentation dieser Liste.
Vllt "as string" ans Ende?

File: dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/services/BulkDataRequestManager.kt [hunk 4]
```diff
@@ -77,21 +77,7 @@ class BulkDataRequestManager(
...
         val retrievedStoredDataRequestsForUser = retrievedStoredDataRequestEntitiesForUser.map { dataRequestEntity ->
-            StoredDataRequest(
-                dataRequestEntity.dataRequestId,
-                dataRequestEntity.userId,
-                dataRequestEntity.creationTimestamp,
-                utils.getDataTypeEnumForFrameworkName(dataRequestEntity.dataTypeName),
-                dataRequestEntity.reportingPeriod,
-                dataRequestEntity.dataRequestCompanyIdentifierType,
-                dataRequestEntity.dataRequestCompanyIdentifierValue,
-                objectMapper.readValue(
```
Comment (emanueldfine) [inline dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/services/BulkDataRequestManager.kt:88]: Wieso ist die messageHistory eigentlich nicht einfach nullable?
Würde man sich dadurch nicht eine Menge Ärger ersparen, weil man dann nicht immer leere Listen da reinpacken müsste?

Und warum ist "dataType" eigentlich nullable in "StoredDataRequest" ?
Sollte nicht jeder data request einen eindeutigen dataType-Wert haben?
Immerhin ist die Kombination dataType-reportingPeriod-companyId-userId der primary key zu jeder einzelnen stored datarequest oder?

File: dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/utils/DataRequestManagerUtils.kt (hunk 3 above)
Comment (fsoffel) [reply dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/utils/DataRequestManagerUtils.kt:31]: Jups, wollte ich. Hab mich dann aber im Namen verloren :D

File: dataland-community-manager/src/main/resources/email-templates/claim_ownership.html.ftl (hunk 1 above)
Comment (fsoffel) [reply dataland-community-manager/src/main/resources/email-templates/claim_ownership.html.ftl:4]: Should exist already

File: dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/utils/DataRequestManagerUtils.kt (hunk 3 above)
Comment (fsoffel) [reply dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/utils/DataRequestManagerUtils.kt]: Längst erledigt. Upsi.

**Pull request merged**